Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.0.0'
+gem 'rails', '~> 5.0.1'
```

Comment:
Yeah, was getting a weird bug in ActiveRecord - I'll check specifics if you want, don't remember offhand.  Stackoverflow said the bug was fixed in 5.0.1